keyboards/ymd96/keymaps/AnthonyWharton/keymap.c

```diff
@@ -0,0 +1,68 @@
+/*
+Base Copyright 2017 Luiz Ribeiro <luizribeiro@gmail.com>
+Modified 2017 Andrew Novak <ndrw.nvk@gmail.com>
+Modified 2018 Anthony Wharton <th3ant@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public LicensezZZ
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+#define ______ KC_TRNS
```

Comment:
Got it, will take a moment to reformat. I was just following [the documentation here](https://beta.docs.qmk.fm/detailed-guides/keymap#anatomy-of-a-keymap-c) which defines a similar macro, which may or may not be out of date? I did wonder why there was conflicting info, but I'll switch it over. If that documentation is out of date, where could I submit a change to this?

Reply:
Yep, I just looked at the edit log for that file and the recommendations actually changed since the last time that doc was edited. So it's out-of-date.

That particular docs lives at [docs/keymap.md](https://github.com/qmk/qmk_firmware/blob/48a992f1c037658bbacccefd2709ffdcda8bb345/docs/keymap.md), but any changes to it we would want in a separate pull request.